apps/frontend/src/components/cards/controltable/ControlRowHeader.vue

```diff
@@ -263,5 +268,22 @@ export default class ControlRowHeader extends mixins(HtmlSanitizeMixin) {
       return {label: cci, url: '', description: this.descriptionForTag(cci)};
     });
   }
+  showLegacy(control: any) {
+    const legacyTag = control.data.tags['legacy'];
+    if (legacyTag) {
```

Comment:
@HenryXiaoHX usually it's better to have 'guard clauses' with early outs so that you can avoid nesting conditionals and have 'else' statements that are way far away from the relevant logic esp when the 'else' stuff is just like 'return default case'.  what you can do instead is make your conditional the opposite (so in this case '!legacyTag'), do the relevant else content (early out with empty string), and now the rest of the function is not indented one level.